### ISSUE TITLE
Support FVDM for saving training time and better results

### ISSUE THREAD

**Issue by sdbds** [kohya-ss/musubi-tuner#220]
source:https://github.com/Yaofang-Liu/Pusa-VidGen

Core:
sampling and applying independent noise levels (sigma) for each temporal position (latent_t) in the latent video.

在图像模型的时代，我们一般默认使用1000timestpes，然而视频出现了帧这额外的维度。

这个方法使用随机

latent_t sample part:

https://github.com/Yaofang-Liu/Mochi-Full-Finetuner/blob/1762e96cfb154b35734d79e4d8322f3ac9789821/src/genmo/mochi_preview/pipelines_ti2v_release.py#L333-L395

random steps for different sigma:

https://github.com/Yaofang-Liu/Mochi-Full-Finetuner/blob/1762e96cfb154b35734d79e4d8322f3ac9789821/src/genmo/mochi_preview/train_pusa.py#L234-L292

`random_steps = torch.randint(0, num_total_steps - 1, (B, latent_t), device=z_0.device) `:
This is the core implementation of the Vectorized Timestep Variable (VTV) concept from FVDM. Instead of sampling a single random step/timestep t for the entire video clip (B,), it samples an independent random step for each temporal position in the latent video, resulting in a shape (B, latent_t).
`sigma = sigma_schedule[random_steps`]: 
Uses the sampled random_steps to look up the corresponding noise levels sigma from the pre-defined schedule. sigma now also has the shape (B, latent_t), meaning each latent frame gets its own noise level based on its sampled step.
`next_sigma, dsigma`: 
Related values from the schedule, potentially for specific SDE solvers or loss weightings (though dsigma isn't used in the loss shown).

**Comment by Sarania** [comment]
Oh this is super duper interesting. I was reading their paper and I can definitely see how this could enhance the model's understanding of time based concepts. I'm excited to see this in action!
